docs: Update install instructions (#1284)
farcasterxyz/hub-monorepo: apps/hubble/www/docs/intro/install.md

- If upgrading from 1.3.3 or below, please set `ETH_MAINNET_RPC_URL=your-ETH-mainnet-RPC-URL` (if using docker) or provide the `--eth-mainnet-rpc-url` flag (if not using docker)

> - If you're changing your Hub from one network to another, you'll need to reset your database. WARNING: This will delete your DB  

Reviewer: ```suggestion
- If you're changing your Hub from one network to another, you'll need to delete your database contents: 
```
